hw/ip/rv_plic/doc/checklist.md

```diff
@@ -0,0 +1,214 @@
+[REVIEW_TODO]:         {{<relref "/doc/project/checklist.md#review-todo" >}}
+[STYLE_X]:             {{<relref "/doc/project/checklist.md#style-x" >}}
+[LINT_PASS]:           {{<relref "/doc/project/checklist.md#lint-pass" >}}
+[CDC_SETUP]:           {{<relref "/doc/project/checklist.md#cdc-setup" >}}
+[CDC_SYNCMACRO]:       {{<relref "/doc/project/checklist.md#cdc-syncmacro" >}}
+[FPGA_TIMING]:         {{<relref "/doc/project/checklist.md#fpga-timing" >}}
+
+### D3
+
+ Type         | Item                    | Resolution  | Note/Collaterals
+--------------|-------------------------|-------------|------------------
+Documentation | [NEW_FEATURES_D3][]     |             |
+RTL           | [TODO_COMPLETE][]       |             |
+Code Quality  | [LINT_COMPLETE][]       |             |
+Code Quality  | [CDC_COMPLETE][]        |             |
+Review        | [REVIEW_RTL][]          |             |
+Review        | [REVIEW_DELETED_FF][]   |             |
+Review        | [REVIEW_SW_CSR][]       |             |
+Review        | [REVIEW_SW_FATAL_ERR][] |             |
+Review        | [REVIEW_SW_CHANGE][]    |             |
+Review        | [REVIEW_SW_ERRATA][]    |             |
+Review        | Reviewer(s)             |             |
+Review        | Signoff date            |             |
+
+[NEW_FEATURES_D3]:      {{<relref "/doc/project/checklist.md#new-features-d3" >}}
+[TODO_COMPLETE]:        {{<relref "/doc/project/checklist.md#todo-complete" >}}
+[LINT_COMPLETE]:        {{<relref "/doc/project/checklist.md#lint-complete" >}}
+[CDC_COMPLETE]:         {{<relref "/doc/project/checklist.md#cdc-complete" >}}
+[REVIEW_RTL]:           {{<relref "/doc/project/checklist.md#review-rtl" >}}
+[REVIEW_DBG]:           {{<relref "/doc/project/checklist.md#review-dbg" >}}
+[REVIEW_DELETED_FF]:    {{<relref "/doc/project/checklist.md#review-deleted-ff" >}}
+[REVIEW_SW_CSR]:        {{<relref "/doc/project/checklist.md#review-sw-csr" >}}
+[REVIEW_SW_FATAL_ERR]:  {{<relref "/doc/project/checklist.md#review-sw-fatal-err" >}}
+[REVIEW_SW_CHANGE]:     {{<relref "/doc/project/checklist.md#review-sw-change" >}}
+[REVIEW_SW_ERRATA]:     {{<relref "/doc/project/checklist.md#review-sw-errata" >}}
+
+## Verification Checklist
+
+### V1
+
+ Type         | Item                                  | Resolution  | Note/Collaterals
+--------------|---------------------------------------|-------------|------------------
+Documentation | [DV_PLAN_DRAFT_COMPLETED][]           | Done        | [rv_plic_fpv_plan]({{<relref "dv_plan/" >}})
+Documentation | [TESTPLAN_COMPLETED][]                | Done        |
+Testbench     | [TB_TOP_CREATED][]                    | Done        | FPV testbench created
+Testbench     | [PRELIMINARY_ASSERTION_CHECKS_ADDED][]| Done        |
+Testbench     | [TB_ENV_CREATED][]                    | N/A         |
+Testbench     | [RAL_MODEL_GEN_AUTOMATED][]           | Done        | FPV CSRs assertion model
```

Comment:
@sriyerg @msfschaffner do you two think it is okay to specify FPV here or shall we change the checklist to explain both DV and FPV scenarios?

Reply:
I think we need a separate checklist for FPV. WDYT?

Reply:
Thanks for the suggestions. So for modules like alert_handler that has both FPV and DV tbs, do you think we are going to list:
```
### V1 for DV
### V1 for FPV
### V2 for DV
### V2 for FPV
```

Reply:
That sounds wise: create two cut/paste template items: one for DV, one for FPV. Would V3 be the same for both?

Reply:
Sorry I did not include the entire list, V3 should be different as well.
```
### V1 for DV
### V1 for FPV
### V2 for DV
### V2 for FPV
### V3 for DV
### V3 for FPV
```

Reply:
+1 for listing them separately

Reply:
We will continue to have our existing milestone checklists V1, V2 and V3 that combines both, sim and FPV specific work items. We just clearly indicate which items are specific to sim and which ones are specific to FPV for each milestone (@cindychip's PR #1481). Based on whether an IP has SIM, FPV or both, items that are not applicable are marked 'NA'. If an IP has both types of testbenches (alert handler), then V2 (for example) should indicate completion of work items for that milestone in both of those types of testbenches. A totally separate checklist seemed unnecessary. DV testbench and FPV testbench can independently make progress; we mark completion of a milestone when the one that is lagging behind catches up.

Reply:
Ok, that makes sense, thanks for clarifying. Important is that the DV/FPV items are distinguishable.